README.md

```diff
@@ -21,7 +21,7 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+numbers at the beginning is timestamp. its a migration file that is generated to add new table in the database.
 ```
```

Comment:
The numbers are a timestamp, but they serve a particular purpose: to let the migration know when it should be run relative to other migrations. Once a migration is run, we don't want it to be run again (unless the database has been dropped or some similar scenario)!